app/lib/efile/ny/it201.rb

```diff
@@ -420,39 +432,36 @@ def calculate_line_69
         # federal adjusted gross income from Form IT-201, line 19, minus distributions from an individual retirement
         # account and an individual retirement annuity, from Form IT-201, line 9, if they were included in your federal
         # adjusted gross income.
-        if (line_or_zero(:IT201_LINE_19) - line_or_zero(:IT201_LINE_9)) < 250_000 && @intake.nyc_full_year_resident_yes?
-          if @filing_status.in?([:single, :married_filing_separately, :head_of_household])
-            63
-          else
-            125
-          end
-        else
-          0
-        end
+        income_eligible = (line_or_zero(:IT201_LINE_19) - line_or_zero(:IT201_LINE_9)) < 250_000
+        # TODO: The instructions provide a credit to part-year nyc residents based on number of months in NYC - should we support that?
```

Comment:
We currently screen out any part-year NYC residents, so I assume that we won't support that.